frog/imports/client/Wiki/index.js

```diff
@@ -654,13 +657,15 @@ class WikiComp extends React.Component<WikiCompPropsT, WikiCompStateT> {
       <div style={sideNavBarStyle}>
         <h2>{this.wikiId}</h2>
         <ul>
-          <Button
-            variant="contained"
-            color="primary"
-            onClick={() => this.setState({ createModalOpen: true })}
-          >
-            + Create new page
-          </Button>
+          {(!this.state.settings?.limitPageCreation || this.state.isOwner) && (
```

Comment:
`limitPageCreation` is not defined in `WikiSettingsT`

Reply:
Made the respective changes